ps_imageslider.php

```diff
@@ -151,7 +151,7 @@ protected function installSamples()
                 $slide->url[$language['id_lang']] = 'http://www.prestashop.com/?utm_source=back-office&utm_medium=v17_homeslider'
                     . '&utm_campaign=back-office-' . Tools::strtoupper($this->context->language->iso_code)
                     . '&utm_content=' . (defined('_PS_HOST_MODE_') ? 'ondemand' : 'download');
-                $slide->image[$language['id_lang']] = 'sample-' . $i . '.jpg';
+                $slide->image[$language['id_lang']] = 'sample-' . $i . ($language['is_rtl'] ? '-rtl' : '') . '.jpg';
```

Comment:
This piece of code looks absolutely correct, just a bit uneasy to read in my humble opinion:
```suggestion
                $slide->image[$language['id_lang']] = implode([
                     'sample-',
                     $i,
                     $language['is_rtl'] ? '-rtl' : '',
                     '.jpg',
                ]);
```